Duplicati/UnitTest/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NUnit" version="3.12.0" targetFramework="net471" />
-</packages>
+  <package id="NUnit3TestAdapter" version="4.2.1" targetFramework="net471" />
```

Comment:
This is a unit test change, that I think we should have.  If we are doing it, can we use the latest?

Reply:
Good catch - 4.2.1 was the latest when I authored this commit (several months ago).
Added a new commit that bumps both NUnit3TestAdapter and NUnit to their latest (stable) versions - 4.4.2, 3.13.3 respectively.